src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/troubleshooting/missing-logs.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Troubleshoot OpenTelemetry: Missing logs in distributed tracing"
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Troubleshooting
+metaDescription: Troubleshoot tips for when you use OpenTelemetry with New Relic
+freshnessValidatedDate: 2024-05-01
+---
+
+## Problem
+
+You want to see logs associated with traces, so you've correlated your logs with your [`service.name`](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs), but you don't see logs in the New Relic UI. In this scenario, the log data has made it to New Relic, but isn't showing up in the distributed trace UI with corresponding spans.
+
+## Solution
+
+To correlate your logs with trace data, the logs need to include the trace context in `trace_id` and `span_id`. However, to ensure your logs show up in the New Relic UI, you'll need to configure rules in your log pipeline to translate `trace_id` and `span_id` to `trace.id` and `span.id`.
```

Comment:
How do they do this? Is there an example we can provide?

Reply:
This document lacks a lot of context. That is, this is not a general problem people will experience. It is a problem that may occur in specific use cases. We can elaborate and improve this doc in a separate PR.

That said, we have an example here. https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/java/logs-in-context-log4j2#introduction.